library/std/src/sys/unix/ext/fs.rs

```diff
@@ -8,6 +8,9 @@ use crate::path::Path;
 use crate::sys;
 use crate::sys::platform::fs::MetadataExt as UnixMetadataExt;
 use crate::sys_common::{AsInner, AsInnerMut, FromInner};
+// Used for `File::read` on intra-doc links
```

Comment:
This is a intra-doc link bug?

Reply:
No, associated items from traits are only resolved if the trait is in scope. It's the same rules as normal path resolution, although I don't think it's documented anywhere but https://github.com/rust-lang/rfcs/blob/master/text/1946-intra-rustdoc-links.md#traits-in-scope. If you're interested in documenting it in the rustdoc book that would be very helpful :)

Reply:
When importing them as `Read as _`, does it work ? It would avoid the `#[allow(unused_imports)]` I thinK.

Reply:
The links resolve but it doesn't allow removing `unused_imports`. Let me know if you still want me to switch.

Reply:
If it doesn't remove the attribute don't bother, this is clear and less cluttery this way